### ISSUE TITLE
Interface SearchOptions incorrectly extends interface Timeoutable

### ISSUE THREAD

**Issue by devinschulz** [mailosaur/cypress-mailosaur#18]
I noticed this error today after upgrading some of my project's dependencies: 

```
node_modules/cypress-mailosaur/src/mailosaurCommands.d.ts:286:18 - error TS2430: Interface 'SearchOptions' incorrectly extends interface 'Timeoutable'.
  Property 'timeout' is optional in type 'SearchOptions' but required in type 'Timeoutable'.

286 export interface SearchOptions extends Cypress.Timeoutable {
                     ~~~~~~~~~~~~~


Found 1 error.
```

* `cypress-mailosaur@2.3.0`
* `cypress@7.2.0`

**Comment by hippyhorse** [comment]
Hey, thanks for this, we'll get it looked into and fixed as soon as possible 👍

**Comment by jhendershott** [comment]
Just putting in a reproduced vote - 
```
cypress-mailosaur:2.3.2,
cypress: 7.5.0,
```

**Comment by efreethy** [comment]
Is there a stop gap solution for this?

**Comment by devinschulz** [comment]
@efreethy Not ideal, but I got around this by skipping type-checking the node modules folder. You can do this by adding `{ "compilerOptions": { "skipLibCheck": true }}` to your `tsconfig.json` https://www.typescriptlang.org/tsconfig/#skipLibCheck

**Comment by andosaur** [comment]
> Is there a stop gap solution for this?

No need anymore, this issue has been fixed in the latest 2.3.3 release.
Thanks for raising this issue and let us know if you encounter any issues either via support or here.
